main.py

```diff
@@ -84,10 +108,88 @@
             # The other members are given access through another Mailbox.
             # You could make the work easier if you also posted a cc channel message already over here.
 
-            # TODO
+            if ' ' not in element.name:
+
+                if element.owner not in element.members:
+                    element.members.append(element.owner)
+                for buddy in element.settlers:
+                    if buddy not in element.members:
+                        msg = """**Warning:** I'm adding settlers to a channel!\nThis is should not be a problem, \
+                        but it does at least indicate a flaw in the bot's code. Please, report this to the Game Masters!"""
+                        await client.get_channel(message.channel).send(msg)
+                        element.members.append(buddy)
+
+                intro_msg = creation_messages.cc_intro(element.members)
+
+                viewers = []
+                abductees = []
+                for member in db.player_list():
+                    if db_get(member,'abducted') == 1:
+                        abductees.append(member)
+                    else:
+                        if member in element.members or db_get(member,'role') in ['Dead','Spectator']:
+                            viewers.append(member)
+
+                # Role objects (based on ID)
+                main_guild = botspam_channel.guild # Find the guild we're in
+                roles = main_guild.roles # Roles from the guild
+                game_master_role = discord.utils.find(lambda r: r.id == game_master, roles)
+                dead_participant_role = discord.utils.find(lambda r: r.id == dead_participant, roles)
+                frozen_participant_role = discord.utils.find(lambda r: r.id == frozen_participant, roles)
+                default_permissions = {
+                    main_guild.default_role: discord.PermissionOverwrite(read_messages=False),
+                    frozen_participant_role: discord.PermissionOverwrite(send_messages=False),
+                    dead_participant_role: discord.PermissionOverwrite(read_messages=True, send_messages=False),
+                    game_master_role: discord.PermissionOverwrite(read_messages=True),
+                    client.user: discord.PermissionOverwrite(read_messages=True,send_messages=True),
+                    **{
+                        member: discord.PermissionOverwrite(read_messages=True) for member in viewers
+                    },
+                }
+
+                # Create a new category if needed
+                if db.get_category() == None:
+                    category = await main_guild.create_category('CC part {}'.format(db.count_categories(), reason='It seems like we couldn\'t use our previous category! Don\'t worry, I just created a new one.')
```

Comment:
Audit log message provides no info and seems unnecessary.

Reply:
db.get_category() always returns None, regardless of whether a category is full, doesn't exist or couldn't be found. The reason why a new one is created, but I guess it's always better than nothing. ¯\_(ツ)_/¯

Reply:
You dropped this \

To prevent anymore lost limbs throughout Github, correctly escape the arms and shoulders by typing the shrug as ¯\\\_(ツ)_/¯ or ¯\\\_(ツ)\_/¯